### PR TITLE
[WIP] feat: add ability to start js-ipfs nodes

### PR DESCRIPTION
**Pull request by dryajov**
This depends on - https://github.com/ipfs/js-ipfs/pull/1067

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,14 +4,14 @@
...
-    "release-major": "aegir-release --type major --env node --docs",
-    "coverage-publish": "aegir-coverage publish"
+    "lint": "aegir lint",
+    "coverage": "aegir coverage",
+    "test": "aegir test --target node",
+    "docs": "aegir docs",
+    "release": "aegir release -t node",
+    "release-minor": "aegir release --type minor -t node",
+    "release-major": "aegir release --type major -t node",
+    "coverage-publish": "aegir coverage -u"
```
Comment (diasdavid) [inline package.json]: Please make an aegir upgrade be part of a separate PR

**Comment by dryajov** [comment]
kk, will pull aegir out.

File: package.json (hunk 1 above)
Comment (dryajov) [reply package.json]: kk, will pull aegir out.

**Comment by dryajov** [comment]
this depends on https://github.com/ipfs/js-ipfs/issues/1095 now.

File: test/spawning-daemons.js [hunk 2]
```diff
@@ -0,0 +1,478 @@
...
+            })
+          })
+        })
+
+        // skip on windows for now
+        // https://github.com/ipfs/js-ipfsd-ctl/pull/155#issuecomment-326970190
+        // fails on windows see https://github.com/ipfs/js-ipfs-api/issues/408
+        if (isWindows) {
+          return it.skip('uses the correct ipfs-api')
+        }
```
Comment (thisconnect) [inline test/spawning-daemons.js]: This skip has been removed in https://github.com/ipfs/js-ipfsd-ctl/commit/381ac1c38f0cb7cb19a61b980b1e43844ecb408f, you may have to rebase  https://github.com/ipfs/js-ipfsd-ctl/network

**Comment by diasdavid** [comment]
Is this still needed with https://github.com/ipfs/js-ipfsd-ctl/pull/176 ?

**Comment by dryajov** [comment]
this is not needed anymore - closing.
